Capstone implementation branch

rmiyoyo opened the pull request:
# Work Done
- Created a conference website as part of my capstone project.
- Generated Speakers content dynamically using JavaScript.
- Added linters and configured them to check errors.
- Added loom video for demonstration.
- Deployed website to GitHub pages.

rmiyoyo committed:
Add images to the project

rmiyoyo committed:
Add javascript file to the project

rmiyoyo committed:
Add style to the project

rmiyoyo committed:
Add content to main and about pages

rmiyoyo committed:
Update README to include acknowledgements

rmiyoyo committed:
Add github pages link to readme

rmiyoyo committed:
Add Loom Video presentation link to README

zieeco commented on index.html:
```diff
@@ -0,0 +1,137 @@
+                <li><a><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                <li><a><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a>English</a></li>
+                <li><a>My Page</a></li>
+                <li><a>Logout</a></li>
+              </ul>
+            </div>
+            <div>
+              <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                <ul>
+                  <li><a href="about_page.html" id="about-active"> About</a></li>
+                  <li><a href="#"> Program</a></li>
+                  <li><a href="#"> Join</a></li>
+                  <li><a href="#"> Sponsor</a></li>
+                  <li><a href="#"> News</a></li>
+                  <li class="campaign"><a href="#"> IX Campaign</a></li>
+                </ul>
+              </div> 
+            </div>
+            <nav class="toggle-menu">
+              <div class="mobile-menu">
+                <a href="index.html"><img class="mobile-logo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+              </div>
+              <div class="nav-links">
+                <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+                <ul>
+                  <li><a href="about_page.html" id="about-active">About</a></li>
+                  <li><a href="#">Progress</a></li>
+                  <li><a href="#">Join</a></li>
+                  <li><a href="#">Sponsor</a></li>
+                  <li><a href="#">News</a></li>
+                </ul>
+              </div>
+              <div class="campaign">
+                <ul>
+                  <li><a href="#">Campaign</a></li>
+                </ul>
+              </div>
+            </nav>
+          </header>
```
### CHANGES FOR THE HOME AND ABOUT PAGE


- Hey, great job, however, let's improve our semantic usage on our webpage by moving the header out the 
`<section class="container">` as the header shouldn't be wrapped in this section tag. It's meant to be a standalone element in your webpage,
- Also, the header is the first element in your webpage that holds your introductory content, and the nav tag which the list of your navigation items. please, fix this. :pray:

zieeco commented on about_page.html:
```diff
@@ -0,0 +1,119 @@
+                        <li><a>English</a></li>
+                        <li><a>My Page</a></li>
+                        <li><a>Logout</a></li>
+                    </ul>
+                </div>
+                <div>
+                    <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                        <ul>
+                            <li><a href="about_page.html" id="about-active"> About</a></li>
+                            <li><a href="#"> Program</a></li>
+                            <li><a href="#"> Join</a></li>
+                            <li><a href="#"> Sponsor</a></li>
+                            <li><a href="#"> News</a></li>
+                            <li class="campaign"><a href="#"> IX Campaign</a></li>
+                        </ul>
+                </div>
+            </div>
+            <nav class="toggle-menu">
+                <div class="mobile-menu">
+                    <a href="index.html"><img class="mobile-logo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                </div>
+                <div class="nav-links">
+                    <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+                    <ul>
+                        <li><a href="about_page.html" id="about-active">About</a></li>
+                        <li><a href="#">Progress</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                    </ul>
+                </div>
+                <div class="campaign">
+                    <ul>
+                        <li>
+                            <a href="#">Campaign</a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
```
### CHANGES FOR THE HOME AND ABOUT PAGE


- Hey, great job, however, let's improve our semantic usage on our webpage by moving the header out the 
`<section class="container">` as the header shouldn't be wrapped in this section tag. It's meant to be a standalone element in your webpage,
- Also, the header is the first element in your webpage that holds your introductory content, and the nav tag which the list of your navigation items. please, fix this. :pray:

zieeco commented on about_page.html:
```diff
@@ -0,0 +1,119 @@
+    <section class="container">
+        <header>
+            <div class="top-navigation">
+                <div class="media-links">
+                    <ul>
+                        <li><a><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                        <li><a><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                        <li><a>English</a></li>
+                        <li><a>My Page</a></li>
+                        <li><a>Logout</a></li>
+                    </ul>
+                </div>
```
- Hey, to also enforce good use of the semantic tag, the container for this part of your webpage should be a nav tag as this is the first and top nav tag on your page that contains the list of social media icons. let's improve on this. :pray:

zieeco commented on index.html:
```diff
@@ -0,0 +1,137 @@
+      <section class="container">
+        <header>
+          <div class="top-navigation">
+            <div class="media-links">
+              <ul>
+                <li><a><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                <li><a><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a>English</a></li>
+                <li><a>My Page</a></li>
+                <li><a>Logout</a></li>
+              </ul>
+            </div>
```
- Hey, to also enforce good use of the semantic tag, the container for this part of your webpage should be a nav tag as this is the first and top nav tag on your page that contains the list of social media icons. let's improve on this. :pray:

zieeco commented on index.html:
```diff
@@ -0,0 +1,137 @@
+                <li><a>Logout</a></li>
+              </ul>
+            </div>
+            <div>
+              <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                <ul>
+                  <li><a href="about_page.html" id="about-active"> About</a></li>
+                  <li><a href="#"> Program</a></li>
+                  <li><a href="#"> Join</a></li>
+                  <li><a href="#"> Sponsor</a></li>
+                  <li><a href="#"> News</a></li>
+                  <li class="campaign"><a href="#"> IX Campaign</a></li>
+                </ul>
+              </div> 
```
- Hey, in here also, as this is the second nav tag and not a div, to also enforce good use of the semantic tag, the container for this part of your webpage should also be a nav tag as this is the second nav tag on your page that contains the list of the main page navigation links. let's improve on this. :pray:

zieeco commented on about_page.html:
```diff
@@ -0,0 +1,119 @@
+                        <li><a>Logout</a></li>
+                    </ul>
+                </div>
+                <div>
+                    <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                        <ul>
+                            <li><a href="about_page.html" id="about-active"> About</a></li>
+                            <li><a href="#"> Program</a></li>
+                            <li><a href="#"> Join</a></li>
+                            <li><a href="#"> Sponsor</a></li>
+                            <li><a href="#"> News</a></li>
+                            <li class="campaign"><a href="#"> IX Campaign</a></li>
+                        </ul>
+                </div>
```
- Hey, in here also, as this is the second nav tag and not a div, to also enforce good use of the semantic tag, the container for this part of your webpage should also be a nav tag as this is the second nav tag on your page that contains the list of the main page navigation links. let's improve on this. :pray:

zieeco commented on index.html:
```diff
@@ -0,0 +1,137 @@
+              </ul>
+            </div>
+            <div>
+              <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
```
- Hey, when using the li, it should however be wrapped in the ul tag as the ul tag is used to group the list of unordered list items. Currently, you've used it incorrectly here. Since you're using it just for the purpose of wrapping your logo in a container, you can use the div tag here, as it's appropriate and semantically correct. please, let's fix this.
- Also, note that you committed the closing tag </li>. when updating it to a div tag, kindly remember to add the closing </div> tag.

zieeco commented on about_page.html:
```diff
@@ -0,0 +1,119 @@
+                    </ul>
+                </div>
+                <div>
+                    <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
```
- Hey, when using the li, it should however be wrapped in the ul tag as the ul tag is used to group the list of unordered list items. Currently, you've used it incorrectly here. Since you're using it just for the purpose of wrapping your logo in a container, you can use the div tag here, as it's appropriate and semantically correct. please, let's fix this.
- Also, note that you committed the closing tag </li>. when updating it to a div tag, kindly remember to add the closing </div> tag.

zieeco commented on about_page.html:
```diff
@@ -0,0 +1,119 @@
+            <img src="./images/partners/grand.png" alt="Ford Motors" width="100px" />
+        </div>
+    </section>
+    <section class="footer">
+        <div>
+            <img src="./images/logos/CSi.png" width="30px" />
+        </div>
+        <div>&copy 2023 - InnovationX Summit</div>
+    </section>
```
- Hey, in here, this is the footer of your page which should be a `<footer>` tag and not a `<section>`.  As the footer is meant to be a standalone element in your webpage, you should move it out of this section. kindly update this to improve your semantic usage and to improve your page layout.

zieeco commented on index.html:
```diff
@@ -0,0 +1,137 @@
+              <img src="./images/partners/grand.png" alt="Ford Motors" width="100px" />
+            </div>
+          </section>
+          <section class="footer">
+            <div>
+              <img src="./images/logos/CSi.png" width="30px" />
+            </div>
+            <div>&copy 2023 - InnovationX Summit</div>
+          </section>
```
- Hey, in here, this is the footer of your page which should be a `<footer>` tag and not a `<section>`.  As the footer is meant to be a standalone element in your webpage, you should move it out of this section. kindly update this to improve your semantic usage and to improve your page layout.

zieeco commented on index.html:
```diff
@@ -0,0 +1,137 @@
+            <div>&copy 2023 - InnovationX Summit</div>
+          </section>
+        </section>
+      </section>
```
- Hey, please, kindly remove this extra closed section tag you added, there seems to be no corresponding opening tag for it. let's improve our codebase.

zieeco commented on index.html:
```diff
@@ -0,0 +1,137 @@
+    </head>
+    <body>
+      <section class="container">
+        <header>
```
- Hey, please, let's try to align the top and bottom nav in the header of the desktop version to be on the same vertical line as seen in the images below to improve our design to be close to the main design as much as we can below:

*your design*
![Screenshot from 2023-04-29 19-53-20](https://user-images.githubusercontent.com/59397159/235321100-3c81be04-2195-46be-9ed4-728470ab7d1b.png)

*main design*
![Screenshot from 2023-04-29 20-38-42](https://user-images.githubusercontent.com/59397159/235321195-c2ed8ad4-540b-43e3-aa9d-6aa4a27749f1.png)

zieeco commented on about_page.html:
```diff
@@ -0,0 +1,119 @@
+    </head>
+    <body>
+    <section class="container">
+        <header>
```
- Hey, please, let's try to align the top and bottom nav in the header of the desktop version to be on the same vertical line as seen in the images below to improve our design to be close to the main design as much as we can below:

*your design*
![Screenshot from 2023-04-29 19-53-20](https://user-images.githubusercontent.com/59397159/235321100-3c81be04-2195-46be-9ed4-728470ab7d1b.png)

*main design*
![Screenshot from 2023-04-29 20-38-42](https://user-images.githubusercontent.com/59397159/235321195-c2ed8ad4-540b-43e3-aa9d-6aa4a27749f1.png)

zieeco reviewed (changes requested):
Hi @rmiyoyo,

<img height="230" src="https://media1.giphy.com/media/2yLNN4wTy7Zr8JSXHB/giphy-downsized-medium.gif"/>



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rmiyoyo committed:
Remove header tag from container section.

rmiyoyo committed:
Change footer section tags from section to footer

rmiyoyo committed:
Changed div tag in social media icons to nav tag to enforce good semantics

rmiyoyo committed:
Changed div tag in main menu to nav tag to enforce good semantics

rmiyoyo committed:
Merge branch 'capstone-implementation-branch' of github.com:rmiyoyo/capstone-conference-page into capstone-implementation-branch

rmiyoyo committed:
Remove header from the container section and change div tag to nav tag

rmiyoyo committed:
Fix linter errors in style.css

rmiyoyo commented:
# Work Done After First Review
- Changed ```div``` tags in navigation menus to ```nav``` tags.
- Aligned IX Campaign menu item and top social media links.
- Changed footer tag from ```section``` to ```footer``` as a better semantics practise.
- Enclosed all pending tags.
- Fixed linter errors in CSS file.

Munsa1 reviewed (approved):
Hi @rmiyoyo , 
![congratulations-2](https://user-images.githubusercontent.com/81932478/235324434-7904d617-e4a9-46d2-ab2e-324f902ae407.gif)

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
Implemented Required Changes :heavy_check_mark: 
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
